src/lib/Debridge/contracts.ts

```diff
@@ -0,0 +1,84 @@
+import { ethers } from "ethers";
+import { CHAIN_ID, LISTEN_FOR_TRANSACTIONS } from "../../enums";
+import { IProtocolContractDefinitions } from "../../types";
+import DlnSourceAbi from "./abis/DlnSource.json";
+import DlnDestinationAbi from "./abis/DlnDestination.json";
+
+export enum CONTRACT_ENUM {
+  // The contract used to place orders on Debridge
+  DLN_SOURCE = "DlnSource",
+  DLN_DESTINATION = "DlnDestination",
+}
+
+export enum EVENT_ENUM {
+  // Event for when a order is placed
+  ORDER_PLACED = "0xfc8703fd57380f9dd234a89dce51333782d49c5902f307b02f03e014d18fe471",
+  // Event for when a order is fulfilled
+  ORDER_FULFILLED = "0xd281ee92bab1446041582480d2c0a9dc91f855386bb27ea295faac1e992f7fe4"
+}
+
+export const contracts: IProtocolContractDefinitions = {
+  [CONTRACT_ENUM.DLN_SOURCE]: {
+    interface: new ethers.Interface(DlnSourceAbi),
+    deployments: {
```

Comment:
Please add all the deployment addresses on all the chains. You can use an explorer like routescan that can give you all the transactions for all the chains

https://routescan.io/address/0xeF4fB24aD0916217251F553c0596F8Edc630EB66
![Screenshot 2024-12-25 at 05 38 05](https://github.com/user-attachments/assets/9cb39a14-3d0f-47fb-ba99-2e52d74a4e68)

Reply:
Added